examples/todomvc/src/containers/VisibleTodoList.js

```diff
@@ -2,10 +2,11 @@ import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import * as TodoActions from '../actions'
 import TodoList from '../components/TodoList'
-import { getVisibleTodos } from '../selectors'
+import { getSortedTodos } from '../selectors'
 
 const mapStateToProps = state => ({
-  filteredTodos: getVisibleTodos(state)
+  filteredTodos: getSortedTodos(state),
```

Comment:
Render the list of todos that's been passed through the Wildcard selector for additional sorting + filtering